bc_obps/reporting/api/activity_data.py

```diff
@@ -0,0 +1,24 @@
+from service.activity_service import ActivityService
+from .router import router
+from registration.decorators import handle_http_errors
+from django.http import HttpRequest
+from typing import Tuple
+
+from registration.schema.generic import Message
+from ninja.responses import codes_4xx, codes_5xx
+
+##### GET #####
+
+
+@router.get(
+    "/get-initial-activity-data",
```

Comment:
This should probably be `/initial-activity-data`, the GET is already the HTTP verb